README.md

```diff
@@ -81,7 +105,15 @@ local latency = math.random(1, 10)
 http_requests_latency_hist:observe(latency)
 ```
 
-Using summaries:
+### Summary
+
+Summary samples observations and aggregates them into configurable quantiles.
+Summaries are useful as a service level indicator (e.g. SLAs, SLOs).
+Summary exposed as multiple numerical values:
+- the total count of observed events
+- the total sum of observed values
+- per quantile number of observed events 
```

Comment:
better to swap: `number of observed event per quantile`

Reply:
done